supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from conftest import Fields, TestFields
+from supplier import models
+
+
+class TestSupplierModel:
+    def test_supplier_creation(self, supplier_test):
+        assert supplier_test.user_name == TestFields['USER_NAME_TEST'].value
+        assert supplier_test.first_name == TestFields['FIRST_NAME_TEST'].value
+        assert supplier_test.last_name == TestFields['LAST_NAME_TEST'].value
+        assert supplier_test.password == TestFields['PASSWORD_TEST'].value
+        assert supplier_test.user_type == TestFields['USER_TYPE_TEST'].value
+        assert supplier_test.business_name == TestFields['BUSINESS_NAME_TEST'].value
+
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier_test):
+        supplier_test.save()
+        assert supplier_test in models.Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_supplier(self, supplier_test):
+        supplier_test.save()
+        supplier_test.delete()
+        assert supplier_test not in models.Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_filter_by_field(self, supplier_test):
+        supplier_test.save()
+        for attri, testAttri in zip(Fields, TestFields):
+            print(models.Supplier.filter_by_field(attri.name,
+                                                  testAttri.value))
+        assert True
```

Comment:
What is the point of asserting True? It'll always pass.

Reply:
yea...i dont know why i did that🤷‍♂️

Reply:
or why you kept it and resolved my comment... ? :)

Reply:
ive pushed the new code 6 hours ago, i dont know why it didnt update, any way, pushed it again without `assert true`, and for sure without the print command.